apps/srt-live-transmit.cpp

```diff
@@ -278,6 +282,8 @@ int parse_args(LiveTransmitConfig &cfg, int argc, char** argv)
     cfg.timeout_mode = Option<OutNumber>(params, o_timeout_mode);
     cfg.chunk_size   = Option<OutNumber>(params, "-1", o_chunk);
     cfg.srctime      = Option<OutBool>(params, cfg.srctime, o_srctime);
+    cfg.buffering    = Option<OutNumber>(params, o_buffering);
+    cfg.buffering    = cfg.buffering > 0 ? cfg.buffering : 10;
```

Comment:
Silent rejection of wrong parameters is a bad idea. Rather exit the program with error message (this is still the option checking stage).

Reply:
I agree. Fixed.